pkg/manifests/configfile.go

```diff
@@ -33,8 +33,13 @@ type ConfigFile struct {
 	CommandUpdated *CommandUpdated `yaml:"commandUpdated"`
 	PatchUpdated   *PatchUpdated   `yaml:"patchUpdated"`
 
-	path       string // the absolute path to the .flux.yaml
-	workingDir string // the absolute path to the dir in which to run commands or find a patch file
+	// These are supplied, and can't be calculated from each other
+	originalPath string // the path relative to the repo root, for identifying generated manifests
```

Comment:
I would rename this to be more descriptive. `originalPath` by itself doesn't say much, it took me a while to figure. How about ....

```
filepathRelativeToRepoRoot
filepathAbsolute
workingDirAbsolute
```

In fact, it may be less confusing (in terms of the parameters passed to the constructor) to use the following (all absolute) paths:

```
repoRootDirPath
configPath
workingDir
```

Reply:
> In fact, I think we can leave it as it was
> 
> ```
> path
> workingDir
> ```

Prior to this branch:
 - `workingDir`, as supplied, is the absolute path to the directory.
 - `path` is the absolute path to the config file.

What is lacking there is the path as given by `--git-path` (or implied by its absence), which can't be calculated from those two.

> Isn't what you currently call originalPath the relative path from the workingDir to path?

No, `originalPath` is the path from the root of the git repo to the directory under consideration; i.e., it's `filepath.Rel(root, workingDir)`.

In general you need three bits of information, to be able to report the things described in #2595 as well as process the config file: the absolute base for all the paths, so they can be found in the filesystem; the path under consideration or "working directory"; the path to the config file to use.

But, finally:

> I would rename this to something more descriptive.

Sure, I agree `originalPath` is obscure; I can rearrange or rename them so it's clearer what's going on.